src/client/components/terminal/index.ts

```diff
@@ -331,6 +331,9 @@ export class TerminalView extends LitElement {
   @property({ type: String })
   id!: string
 
+  @property({ type: String })
```

Comment:
@pastuxso I added it to make it reactive since it's validated in line [927].(https://github.com/stateful/vscode-runme/pull/1671/files/364bab4ab6a3a0a55a2b176ee5cc856b59f8bd1c#diff-2af46bd2cfaeee0839a2622a7dff108bb55de11fa71a1d70f984037f99f4a918R927)

Reply:
@pastuxso Good catch. Makes sense!